recipes/ludoSim/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set name = "ludoSim-jaib1" %}
+{% set version = "0.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: e04071d847996209c3e0485dd814207619c3c3e3d404894fa531d03622f8c8d5
+  - url: https://raw.githubusercontent.com/jaib1/ludoSim/master/LICENSE
+    sha256: 3972dc9744f6499f0f9b2dbf76696f2ae7ad8af9b23dde66d6af86c9dfb36986
+
+build: 
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: true  # [py<30]
```

I think this can be ``noarch``
```suggestion
  noarch: python
```